python/auto_sip.blacklist

```diff
@@ -247,6 +247,7 @@ core/symbology-ng/qgssymbollayerregistry.sip
 core/symbology-ng/qgssymbollayerutils.sip
 core/symbology-ng/qgsvectorfieldsymbollayer.sip
 core/symbology-ng/qgsarrowsymbollayer.sip
+core/symbology-ng/qgsconcentriclegendsymbol.sip
```

Comment:
Did you run `./scripts/sipify_all.sh` after removal of these lines in the blacklist?